frontend/src/main/java/puf/frisbee/frontend/viewmodel/GameViewModel.java

```diff
@@ -452,7 +452,11 @@ public void showQuitConfirmDialog() {
 	}
 
 	public void saveGame() {
-		this.teamModel.setLevel(this.levelModel.getCurrentLevel());
+		if (this.teamModel.getLevel() < this.gameModel.getMaximumLevel()) {
+			this.teamModel.setLevel(Integer.parseInt(this.labelLevel.getValue()) + 1);
```

Comment:
Wieso nicht `this.teamModel.setLevel(this.teamModel.getLevel() + 1)`?

Reply:
Müsste ich gleich mal testen.